cmd/lotus/daemon.go

```diff
@@ -184,6 +188,10 @@ var DaemonCmd = &cli.Command{
 			return xerrors.Errorf("repo init error: %w", err)
 		}
 
+		if cctx.String("config") != "" {
+			r.SetConfigPath(cctx.String("config"))
+		}
```

Comment:
Shouldn't this be above r.Init?

Reply:
I don't think it makes a difference? If above Init, we won't create a default config in the default path (if missing), which seems neither good nor bad?